### PR TITLE
Basic app structure prepared.

### LINKED ISSUE #11: Stworzyć podstawową strukturę aplikacji.

### PR DESCRIPTION
**Pull request by krystian8207**
##### Link to backlog task or description:

Closes #11 

Warning is non-skippable: 
```
Warning in loadSupport(appDir, renv = sharedEnv, globalrenv = NULL) :
  Loading R/ subdirectory for Shiny application, but this directory appears to contain an R package. Sourcing files in R/ may cause unexpected behavior.
```

##### Definition of Done:
- [x] All project work has a corresponding task. If there's no task for what you are doing, create it. Each task needs to be well defined and described.
- [x] Change has been tested (manually or with automated tests), everything runs correctly and works as expected. No existing functionality is broken.
- [x] No new error or warning messages are introduced.
- [x] All interaction with a user, user-facing messages, plots, reports etc. are written from the perspective of the person using or receiving it. They are understandable and helpful to this person. If a user sees an error message, there is a call to action, i.e. the user knows what to do to fix it.
- [x] README, other documentation and code comments that we have is updated with all information related to the change.
- [ ] All code has been peer-reviewed before merging into any main branch.
- [ ] All changes have been merged into the main branch we use for development.
- [x] Any added or touched code follows our style-guide.

##### Good practices

* remember to self-review your code
* remember to update credentials in 1password if this is related to this change

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
 Type: Package
-Title: What the Package Does (Title Case)
-Version: 0.1.0
-Author: Who wrote it
-Maintainer: The package maintainer <yourself@somewhere.net>
-Description: More about what it does (maybe more than one line)
-    Use four spaces when indenting paragraphs within the Description.
-License: What license is it under?
+Title: Display Sales Data
+Version: 0.0.0.9003
```
Comment (igrasek) [inline DESCRIPTION]: Why that version? Let's start with sth 1ish.

File: R/output_utils.R [hunk 2]
```diff
@@ -0,0 +1,25 @@
+data_header <- function(store, department) {
```
Comment (igrasek) [inline R/output_utils.R]: Is this file required? I cannot se it to be used anywhere?

**Review by igrasek — CHANGES_REQUESTED** [review]
Ptal at the comments.

File: DESCRIPTION (hunk 1 above)
Comment (krystian8207) [reply DESCRIPTION]: done

**Review by igrasek — APPROVED** [review]
lgtm
